incydr/_users/client.py

```diff
@@ -31,17 +31,29 @@ class UsersV1:
     def __init__(self, parent):
         self._parent = parent
 
-    def get_user(self, user_id: str) -> User:
+    def get_user(self, user_id: str = None, username: str = None) -> User:
```

Comment:
I know this is basically just a wrapper for the `get_page()` username filter but I think its a more obvious place for users to find it.  If we think this is extraneous we can remove it.

Reply:
Why don't we just have this take a single arg `user` that could be either an id or a username, then we don't need the `cli.util.user_lookup` function?

Reply:
This is such a common case we need to do in almost every integration, might as well make it so we don't need to reimplement it 6 different times.

Reply:
I think you're right that we should change `get_user` to take a single arg.  

To the second point - `get_user` returns a `User` object while `user_lookup` returns the `id` and replaces the username value coming from the cli arg.  I also think it helps to have custom help text for each of the user args, so I'm not sure we could get away with defining a  single `@user_option` decorator.

Reply:
Ah, true. I guess if you need an Id for another API call, and you already have an Id, you wouldn't be passing it to this method...

Reply:
yus  🙌